modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsLayoutsDisplayContext.java

```diff
@@ -110,6 +120,32 @@ public long getObjectDefinitionId() {
 		return objectDefinition.getObjectDefinitionId();
 	}
 
+	public List<Map<String, String>> getObjectFieldBusinessTypeMaps(
+		Locale locale) {
+
+		List<Map<String, String>> objectFieldBusinessTypeMaps =
+			new ArrayList<>();
+
+		for (ObjectFieldBusinessType objectFieldBusinessType :
+				_objectFieldBusinessTypeServicesTracker.
+					getObjectFieldBusinessTypes()) {
+
+			objectFieldBusinessTypeMaps.add(
+				HashMapBuilder.put(
+					"businessType", objectFieldBusinessType.getName()
+				).put(
+					"dbType", objectFieldBusinessType.getDBType()
+				).put(
+					"description",
+					objectFieldBusinessType.getDescription(locale)
+				).put(
+					"label", objectFieldBusinessType.getLabel(locale)
+				).build());
+		}
+
+		return objectFieldBusinessTypeMaps;
+	}
+
```

Comment:
@marcelabc can you create a Util class (or create a BaseObjectFieldDisplayContext) to isolate this piece of code? And also adapt to the changes that Carolina did in the other method?

Reply:
@gabrielwas We don't need to adapt to the changes that Carolina did because this piece of code won't show anything to the user. It is only to match the field already created by the user in order to show the translated Label.

The other *DisplayContext is used to show the options to the user, so it needs to filter specifics ObjectFieldBusinessType.